<!DOCTYPE html>
<html lang="en">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src="https://code.jquery.com/jquery-1.10.2.js"></script>


    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
    <!--Font Awesome-->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
    <!--css-->
    <link href="styletest.css" rel="stylesheet">

    <!--fonts-->
    <link href="https://fonts.googleapis.com/css?family=Montserrat|Oswald" rel="stylesheet">
    <!--jquery-->


    <title>Experiments</title>

    <style>

      .card .card-header {
        padding: 0;

      }

      img {
        max-width: 100%;
      }



      .card .card-header .btn {
        font-size: 20px;
        font-weight: 600;
        color: black;
      }

      .card .card-header h2 {
        padding: 20px;
      }


      .card h4 {
        color: black;
      }
      .row {
        margin-top: auto;
      }


      h1,h2,h3,h4 {
        font-family: 'Oswald', sans-serif;
        text-align: left;
      }

      p.larger {
        font-size: 140%;
      }

      .text-highlight-box {
        background: white;
        padding: 20px;
        font-size: 80%;
        border: 1px solid #ccc;
      }

    </style>

  </head>
  <body>
    <!--TOP LOGO-->
    <div class="header">
      <img src="logos/vibx/vibx_tp1.png" style="width: 100%; margin-top: 0%; background-position: center; left: 0; right: 0;">
    </div>

    <!--SCROLL UP BUTTON-->
    <button onclick="topFunction()" id="myBtn" title="Go to top"><i class="fas fa-arrow-circle-up"></i></button>

    <script>
      // When the user scrolls down 20px from the top of the document, show the button
      window.onscroll = function() {scrollFunction()};

function scrollFunction() {
  if (document.body.scrollTop > 200 || document.documentElement.scrollTop > 200) {
    document.getElementById("myBtn").style.display = "block";
  } else {
    document.getElementById("myBtn").style.display = "none";
  }
}

// When the user clicks on the button, scroll to the top of the document
function topFunction() {
  document.body.scrollTop = 0;
  document.documentElement.scrollTop = 0;
}
    </script>


    <div class="progress-container">
      <div class="progress-bar" id="myBar"></div>
    </div>

    <script>
      // When the user scrolls the page, execute myFunction
      document.onscroll = function() {myProgress()};

function myProgress() {
  var winScroll = document.body.scrollTop || document.documentElement.scrollTop;
  var height = document.documentElement.scrollHeight - document.documentElement.clientHeight;
  var scrolled = (winScroll / height) * 100;
  document.getElementById("myBar").style.width = scrolled + "%";
}
    </script>


    <!--------------------------------------------------------TOP MENU----------------------------------------------------------------->


    <div id="topnav">
      <nav>
        <ul id="menu">
          <li><a href="#">Home</a></li>

          <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Description">Project</a>

            <ul class="submenu" style="margin: 0 0 0 0;">
              <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Description">Description</a></li>
              <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Design">Design</a></li>
              <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Notebook">Notebook</a></li>
            </ul>
          </li>

          <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Lab">Lab</a>
            <ul class="submenu" style="margin: 0 0 0 0;">
              <li style="font-size: 20px; padding: 5px 10px 0px 10px; margin-bottom: 0;">Wet Lab
                <ul class="submenu" style="margin: 0 0 0 0;">
                  <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Safety">Safety</a></li>
                  <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Experiments">Experiments</a></li>
                  <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Demonstrate">Results</a></li>
                </ul>
              </li>
              <li style="font-size: 20px; padding: 5px 10px 0px 10px; margin-bottom: 0;">Dry Lab
                <ul class="submenu" style="margin: 0 0 0 0;">
                  <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Model">Model overview</a></li>
                  <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Model/Simulations">Simulations</a></li>
                </ul>
              </li>
            </ul>
          </li>

          <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Parts">Parts</a>
            <ul class="submenu" style="margin: 0 0 0 0;">
              <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Basic_Part">Basic Parts</a></li>
              <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Composite_Part">Composite Parts</a></li>
              <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Improve">Improved parts</a></li>
              <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Part_Collection">Collection</a></li>
            </ul>
          </li>

          <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Human_Practices/Overview">Human Practices</a>
            <ul class="submenu" style="margin: 0 0 0 0;">
              <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Public_Engagement">Education & Public Engagement</a></li>
              <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Collaborations">Collaborations</a></li>
              <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Human_Practices">Integrated Human Practices</a>
                <ul class="submenu" style="margin: 0 0 0 0;">
                  <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Entrepreneurship">Entrepreneurship</a></li>
                </ul>
              </li>
            </ul>
          </li>

          <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Team">Team</a>
            <ul class="submenu" style="margin: 0 0 0 0;">
              <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Attributions">Attributions</a></li>
              <li style="margin-bottom: 0;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Sponsors">Sponsors</a></li>
            </ul>
          </li>

          <li id="mainmenu"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Awards">Awards</a></li>

        </ul>
      </nav>
    </div>


    <!--------------------------------------------------------CONTENT----------------------------------------------------------------->


    <!--CONTENT-->

    <div class="content hometext">
      <div class="container-fluid mb-5">
        <div class="hometext">
          <div class="row justify-content-center">
            <div class="col-md-10">
              <div class="row align-items-center">
                <div class="col-md-6">
                  <h1>EXPERIMENTS</h1>
                  <p>Intro text</p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div class="container-fluid">
        <div class="row justify-content-center">
          <div class="col-md-10">
            <div class="accordion" id="accordionExample">
              <div class="card">
                <div class="card-header" id="headingOne">
                  <h2 class="mb-0">
                    <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
                      VIBRIO NATRIEGENS ATCC 14048 / DSM 759 AND VMAX STRAINS
                    </button>
                  </h2>
                </div>

                <div id="collapseOne" class="collapse " aria-labelledby="headingOne" data-parent="#accordionExample">
                  <div class="card-body">
                    <div class="row mb-1">
                      <div class="col">
                        <p>
                        We obtained our V. natriegens ATCC DSM 759 strain (WT) from DSMZ, but before work with it began, we established basic workflows, protocols (see Protocols), growth mediums, and other materials by working with Synthetic Genomic’s Vmax strain. Vmax is a commercial Vibrio natriegens strain, with an extracellular nuclease knockout and IPTG inducible T7 RNA Polymerase addition (Synthetic Genomics, 2017). After this, we released the WT strain and started to work with it, while maintaining the Vmax strain in parallel in some of the experiments.
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>

            <div class="card">
              <div class="card-header" id="headingTwo">
                <h2 class="mb-0">
                  <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
                    DNA SEQUENCES
                  </button>
                </h2>
              </div>
              <div id="collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordionExample">
                <div class="card-body">
                  <div class="row mb-5">
                    <div class="col">
                      <p>
                      To search for putative signal sequences from V. natriegens DSM 759 strain, we performed a BLAST search based on the S/T-R-R-x-F-L-K consensus sequence usually found in leader peptides of Tat pathway targeted proteins (Lee et al., 2006). The results were then analysed for signal peptides using Phobius (http://phobius.sbc.su.se), and the most promising signal sequence from trimethylamine N-oxide reductase (ssTorA), was chosen for experiments. A homologous version of ssTorA in E. coli is well known to function with the Tat pathway of the respective bacterium (Blaudeck <i>et al.</i>, 2001; Lee <i>et al.</i>, 2006; Alanen <i>et al.</i>, 2015).
                      </p>
                    </div>
                  </div>
                  <div class="row mb-5">
                    <div class="col">
                      <p>
                      Synthetic DNA fragments with <i>ssTorA</i> and other putative signal peptides attached to the YGFP and human growth hormone (hGH) were ordered from Twist DNA. All sequences ordered and received are listed below. Originally we designed the sequences with restriction enzyme cut sites that would enable us to clone them into pQE-30 vector. However, this was changed when plans improved, and we started using PCR to change the cut sites to suit our cloning needs for pC201 and pC201 backbones.
                      </p>
                      <!-- ADD PHOTO -->
                      <!-- https://drive.google.com/drive/folders/1wQKSvtq1Ltss5Kuj7hCmnRBmbsEAi2YK -->
                    </div>
                  </div>
                </div>
              </div>
            </div>


            <div class="card">
              <div class="card-header" id="headingThree">
                <h2 class="mb-0">
                  <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
                    CLONING
                  </button>
                </h2>
              </div>
              <div id="collapseThree" class="collapse" aria-labelledby="headingThree" data-parent="#accordionExample">
                <div class="card-body">
                  <div class="row mb-4">
                    <div class="col">
                      <p>
                      Plasmids pC203 and pC201 were chosen for expression tests because of their availability but also characteristics: they harbored the p15A ori which was known to work in <i>V. natriegens</i> with quite low copy number, had suitable antibiotic resistance for selection, and included inducible, tightly controlled promoters. We believed that these features would allow us to experiment with different genes and expression levels in a not so known host organism.
                      </p>
                    </div>
                  </div>
                  <div class="row mb-5">
                    <div class="col">
                      <p>
                      The original pC203 we received from Christopher Jonkergouw contained a pluR insert without suitable restriction sites for removal. The pC203 backbone was PCR’d using a forward primer annealing to the MCS and a reverse primer containing a 5’-overhang with PstI and EcoRI restriction sites annealing before the pluR-gene. Thus we had created a easy-to-use expression backbone for our gene constructs.
                      </p>
                    </div>
                  </div>
                  <div class="row mb-4">
                    <div class="col">
                      <p>
                      The PCR for pC203 was performed  with our designed primers PC20x_dPluR/dProm fwd & PC203_dPluR rev (anneal T= 62.3C) to remove the unnecessary CDS <i>PluR</i> , and confirmed with electrophoresis gel that we had successfully performed PCR (pC203 without PluR = 4880bp). We also designed and performed <i>PluR</i> removal for plasmid pC201. PCR was performed with Thermo Fisher Scientific’s Phusion polymerase.
                      </p>
                    </div>
                  </div>
                  <div class="row mb-4">
                    <!-- ADD PHOTO -->
                    <!-- https://drive.google.com/drive/folders/1-k1D_npStasS5QDc9LQx10_deuf1TSsI -->
                    <!-- https://drive.google.com/drive/folders/1-k1D_npStasS5QDc9LQx10_deuf1TSsI -->
                  </div>
                  <div class="row mb-4">
                    <div class="col">
                      <p>We successfully performed PCR and confirmed results with an electrophoresis gel showing a band at 690 bp for tfoX amplification</p>
                      <small>Image: Insertion sequence confirmed pC203-ssTorA-YGFP plasmid chart (generated by Benchling).</small>
                      <!-- ADD PHOTO -->
                      <!-- https://drive.google.com/drive/folders/1K_q4cQBim00xcpp555bNoZrE7bX8KysI -->
                      <small>Image: Insertion sequence confirmed pC203-ssTorA-hGH plasmid chart (generated by Benchling).</small>
                      <!-- ADD PHOTO -->
                      <!-- https://drive.google.com/drive/folders/1K_q4cQBim00xcpp555bNoZrE7bX8KysI -->
                      <small>
                        Image: pC203-TorA-hGH plasmid chart (generated by Benchling).
                      </small>
                      <!-- ADD PHOTO -->
                      <!-- https://drive.google.com/drive/folders/1K_q4cQBim00xcpp555bNoZrE7bX8KysI -->
                    </div>
                  </div>
                  <div class="row mb-4">
                    <div class="col">
                      <p>
                      After, we constructed an expression vector by digesting pC203-dPlur and Tfox with EcoRI & XbaI in separate tubes. We Dephosphorylated the digested pC203 with Thermo Fisher Scientific’s FastAP alkaline phosphatase. Then, we denatured the restriction enzymes and alkaline phosphatase. Ligation of the digested pieces was performed using Thermo Fisher Scientific’s T4 DNA Ligase, and the constructs were transformed into <i>V. natriegens</i> and plated on ampicillin containing V2+LB plate. Following plating, we screened for successful inserts using primers Tfox fwd & PC203_insert_conf rev (product = 690bp, anneal T =59,9C)
                      </p>
                      <!-- ADD PHOTO -->
                      <!-- https://drive.google.com/drive/folders/1-k1D_npStasS5QDc9LQx10_deuf1TSsI -->
                    </div>
                  </div>
                </div>
              </div>
            </div>

            <div class="card">
              <div class="card-header" id="headingFour">
                <h2 class="mb-0">
                  <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
                    TRANSFORMATIONS
                  </button>
                </h2>
              </div>
              <div id="collapseFour" class="collapse" aria-labelledby="headingFour" data-parent="#accordionExample">
                <div class="card-body">
                  <div class="row mb-4">
                    <div class="col">
                      <p>
                      Plasmids were transformed into either chemically competent or electrocompetent cells. We prepared both for <i>E. coli</i> DH5a cells, but only electrocompetent <i>V. natriegens</i> cells. Preparation and transformation protocols are listed in Protocols
                      </p>
                    </div>
                  </div>
                  <div class="row mb-4">
                    <div class="col">
                      <p>
                      Induction of pC203-TorA+YGFP was carried out in Vibrio natriegens (Vmax) and Escherichia coli (DH5a) in order to verify that the plasmid construct was working like expected and producing YGFP. Induced E. coli cells were used for microscopy after the experiment, and localisation of YGFP was observed in some of the cells.
                      </p>
                    </div>
                  </div>
                  <div class="row mb-4">
                    <div class="col">
                      <!-- ADD TEXT -->
                    </div>
                  </div>
                </div>
              </div>
            </div>


            <div class="card">
              <div class="card-header" id="headingFive">
                <h2 class="mb-0">
                  <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#collapseFive" aria-expanded="false" aria-controls="collapseFive">
                    qPCR FOR PLASMID COPY NUMBER QUANTIFICATION
                  </button>
                </h2>
              </div>
              <div id="collapseFive" class="collapse" aria-labelledby="headingFive" data-parent="#accordionExample">
                <div class="card-body">
                  <div class="row mb-4">
                    <div class="col">
                      <p>
                      Quantitative PCR (qPCR) was carried out to determine copy numbers of plasmids/cell in Vibrio natriegens according to a method described by Lee at al. (2006). Absolute quantification method was used. Briefly, the method involves isolating total DNA from the bacterium carrying the plasmid of interest. With two sets of primers, total DNA (genomic and plasmid) from the same sample was used as a template for qPCR. Amplification was targeted at the bla gene (encoding for beta-lactamase) and a housekeeping gene from the chromosome. Here we targeted the housekeeping gene dxs, encoding for 1-deoxy-D-xylulose-5-phosphate synthase, which was also used by Lee et al. (2006). The gene is typically present as one copy in the genome, and thus the amount of dxs gene copies in the sample is equal to the amount of cells. In Vibrio, dxs is present in only one of the two chromosomes, and the same principle can be applied. By dividing the number of plasmid copies by the number of copies of the dxs gene, plasmid copy number/cell (or per gDNA) was calculated.
                      </p>
                      <strong>Plasmids</strong>
                      <p>
                      Quantification was carried out for empty plasmid backbones of pUC18, pQE30 and pC203 plasmids, with pMB1 (derivative), p15A and ColE1 origins of replication respectively. pC203 plasmid was used as an expression vector in our experiments. The copy number retrieved from the experiment was utilized in modeling.
                      </p>
                      <strong>Preparation of the templates</strong>
                      <p>
                      Plasmids were transformed into Vibrio natriegens (Vmax) with electroporation as described under “protocols”. Glyserol stocks were prepared from colonies grown from the transformation plate according to Vmax protocol and stored at - 80 C.
                      (https://www.biocat.com/bc/pdf/Vmax_Express_40026_100517-Rev1.6.pdf). To isolate total DNA templates, cells were regrown on LB+amp (25 ug/ml) plates, and then individual colonies were used for starter cultures of each strain (Vibrio natriegens + pUC18, pQE30 or pC203). Starter cultures were prepared in 5 ml of LB+amp (25 ug/ml) medium, and grown at + 37 C, 220 rpm for 45 min. About 300 ul of each starter culture was then used to inoculate 5 ml of fresh LB+amp (25 ug/ml). Each strain was grown in triplicates. Total DNA was harvested at OD600 = 0.5 (exponential growth phase) for pUC18 and pQE30 harboring strains as suggested in the  protocol by Lee et al. (2006). Vibrio natriegens harboring pC203 was isolated at OD600 = 1, as was done by Tschirhart et al. (2019). Total DNA was extracted with innuPREP Bacteria DNA Kit (Analytik Jena) according to protocol. DNA concentrations were in the range of 1635-2893,6 ng/ul, and were diluted 1:100 and stored at -20 C. 1:100 diluted samples were used as templates.
                      </p>
                      <strong>Preparation of standards</strong>
                      <p>
                      As a standard, a dilution series of TOPO TA vector (Invitrogen) containing dxs gene cloned from Vibrio natriegens genome was used. Plasmid copy numbers were in the range of 6,057x10^9 - 6,057x10^3copies/ul. Preparation of standards is presented under Protocols. Successful cloning of dxs into TOPO TA vector was verified by PCR and sequencing using the same set of primers. TOPO TA vector also contains the bla encoding gene, which was used as a target for amplification from pUC18, pQE30 and pC203 plasmids. The same standard was used as a template for quantifying both dxs and bla.
                      </p>
                      <strong>qPCR assay</strong>
                      <p>
                      qPCR was carried of with SensiFAST™ SYBR® No-ROX Kit (Bioline) according to protocol in 20 ul reactions. Total DNA templates were examined as biological triplicates and two technical replicates. Standards were examined in three technical replicates of each copy number (6,057x10^9 - 6,057x10^3copies/ul). No template controls were included. Bla and dxs targeting primers were used in separate wells, and therefore two master mixes were prepared (containing either bla or dxs targeting primers). Standards and templates were amplified using both sets of primers. The experiment was done using white 96-well Multiplate PCR Plates (Bio-Rad) and CFC Connect Real-Time System (Bio-Rad) machine.
                      </p>
                      <p>
                      Bla primers (from article by Lee et al., 2006)
                      <pre>
                          bla_E.coli_F: 5’-CTA CGA TAC GGG AGG GCT TA-3’
                      </pre>
                      <pre>
                          bla_E.coli_R: 5’-ATA AAT CTG GAG CCG GTG AG-3’
                      </pre>
                      </p>
                      <ol>
                      qPCR program:
                        <li>95 C, 3:00 min (1 cycle)</li>
                        <li>95 C, 0:10 s</li>
                        <li>60 C, 0:35 s</li>
                        <li>72 C, 0:30 s (+ plate read)        GOTO 2, repeat 39 x</li>
                        <li>Melt curve from 65 to 95 C, increment 0.5 C, for 0:05s (+plate read)</li>
                        <li>End</li>
                      </ol>
                      <p>
                      Cq values, melt curves and amplification curves were retrieved from the accompanied program. Melting curves were examined to check specificity of primers. Standard curves were plotted for dxs and bla from Cq values and known copy numbers of each gene. Unknown copy numbers of bla and dxs from total DNA templates were calculated from the equation retrieved from standard curves. Plasmid copy numbers were calculated by dividing the number of bla copies/ul with the number of dxs copies/ul.
                      </p>
                    </div>
                </div>
              </div>
            </div>
            </div>

            <div class="card">
              <div class="card-header" id="headingSix">
                <h2 class="mb-0">
                  <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#collapseSix" aria-expanded="false" aria-controls="collapseSix">
                    CHROMOSONAL EDITING
                  </button>
                </h2>
              </div>
              <div id="collapseSix" class="collapse" aria-labelledby="headingSix" data-parent="#accordionExample">
                <div class="card-body">
                  <div class="row mb-4">
                    <div class="col">
                      <p>
                      To obtain our desired VibXPresso strain, we wanted to introduce chromosomal modifications into <i>Vibrio natriegens</i>. We planned to achieve this via homologous recombination, which in bacteria is a major DNA repair process. It also enables genetic diversity in bacteria. For this, we constructed a pC203-tfoX plasmid where we introduced <i>tfoX</i> gene. <i>tfoX</i> is a regulator of natural competence in <i>V. natriegens</i> (Dalia et al., 2017). This increases the efficiency of homologous recombination, making the editing easier.
                      </p>
                      <p>
                      After the construction of pC203-tfoX, we purified the plasmid and continued with  chromosomal modifications. Our planned modifications included dns and tat deletions for <i>V. natriegens</i>. The <i>dns</i> deletion would increase the transformation efficiency of our strain and a tat deletion strain would be created to act as a control for our translocation studies.
                      </p>
                      <p>
                      For the planned dns deletion, we performed PCR to amplify the upstream region for homologous recombination vector using our designed primers Dns 2.6k upstream fwd & Dns OE cfwd-rev (anneal T= 62.7C), and the downstream region for homologous recombination vector using primers Dns 2.5k downstream rev & Dns OE crev-fwd (anneal T= 62.0C).
                      </p>
                      <p>
                      For the planned tat deletion, we also performed PCR to amplify the upstream region for homologous recombination vector using our designed primers TatO 2.54k upstream fwd & TatO OE cfwd-rev (anneal = 61.3), and the downstream region for homologous recombination vector using primers TatO 2.5k downstream rev & TatO OE crev-fwd (anneal T= 61.4C).
                      </p>
                      <p>
                      Due to time constraints, we decided to continue only with the dns deletion into the next step. In the next step, we combine pieces of upstream and downstream regions by overlap extension PCR (OE-PCR) to form a complete vector (anneal T = 72C). Overlap extension PCR (OE-PCR) is a method to scarlessly join together PCR fragments by utilizing primers containing a 5’-end overlap. This makes the fragments act as primers for each other, combining them in PCR. Normal primer design rules should be followed when designing the 5’-overlaps.
                      </p>
                      <!-- ADD PHOTO -->
                      <small>Overlap extension PCR to delete a DNA sequence from the strand.</small>
                      <p>
                      Next, we performed PCR with our designed primers Dns 2.6k upstream fwd & Dns 2.5k downstream rev to amplify complete vector (anneal T= 62.3C). After, we run an electrophoresis gel of PCR products from and extracted the band at 5000bp. We then transformed the complete vector into (naturally competent, induced Tfox expression plasmid containing) V. natriegens and plated transformed cells on ampicillin + V2 + LB containing plate. Next, we picked colonies and attempted to verify deletion by PCR using primers Dns 3.1k upstream fwd & Dns 3k downstream rev (anneal T = 63). Finally, we analyzed the  product in electrophoresis gel (deletion-positive = 6000bp). The confirmation process was still happening at the time of the wiki freeze, so we don’t have a gel image of the confirmation.
                      </p>
                      <!-- ADD PHOTO -->
                      <!-- https://docs.google.com/document/d/1PE7Vwj8-BITGEf8MFZ2ZMl3NyiGmF-l8hJs-eNR-A2M/edit -->
                    </div>
                  </div>
                </div>
              </div>
            </div>



            <div class="card">
              <div class="card-header" id="headingSeven">
                <h2 class="mb-0">
                  <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#collapseSeven" aria-expanded="false" aria-controls="collapseSeven">
                    CELL FRACTIONING
                  </button>
                </h2>
              </div>
              <div id="collapseSeven" class="collapse" aria-labelledby="headingSeven" data-parent="#accordionExample">
                <div class="card-body">
                  <div class="row mb-4">
                    <div class="col">
                      <p>
                      To test the expression and translocation of ssTorA-hGH, we decided to perform a cell fractionation experiment that would allow us to determine the localisation of leader signal including hGH in <i>V. natriegens</i>, while hGH without the signal peptide would serve as a control. Fractionation divides the cells into periplasmic, cytoplasmic and insoluble (membrane) fractions, that can then be analysed separately.
                      </p>
                      <p>
                      After researching for various fractionation methods, we decided on PureFrac that had been developed and used in fractionation of <i>E. coli</i> cells without detectable contamination between the fractions (Malherbe <i>et al.</i>, 2019). PureFrac uses osmotic shock with MgCl2 for the extraction of periplasmic fraction, and sonication for cell disruption. The fractionation was applied successfully to our cells with few modifications to the original protocol (see Protocols).
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>

            <div class="card">
              <div class="card-header" id="headingEight">
                <h2 class="mb-0">
                  <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#collapseEight" aria-expanded="false" aria-controls="collapseEight">
                    WESTERN BLOT
                  </button>
                </h2>
              </div>
              <div id="collapseEight" class="collapse" aria-labelledby="headingEight" data-parent="#accordionExample">
                <div class="card-body">
                  <div class="row mb-4">
                    <div class="col">
                      <p>
                      For the detection of His tagged hGH, we blotted the different fractions to a nitrocellulose membrane using BioRad Trans-Blot Turbo (see Protocols). Chemiluminescent imaging was performed in BioRad ChemiDoc MP imaging system.
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>


          </div>
      </div>

    </div>
    <div class="container-fluid">
      <div class="row justify-content-center">
        <div class="col-10 mt-5">
          <b>References:</b>
          <p>
          Alanen, H. I., Walker, K. L., Lourdes Velez Suberbie, M., Matos, C. F., Bonisch, S., Freedman, R. B., . . . Robinson, C. (2015). Efficient export of human growth hormone, interferon alpha2b and antibody fragments to the periplasm by the Escherichia coli Tat pathway in the absence of prior disulfide bond formation. Biochim Biophys Acta, 1853(3), 756-763. doi:10.1016/j.bbamcr.2014.12.027
          </p>
          <p>
          Blaudeck, N., Sprenger, G. A., Freudl, R., & Wiegert, T. (2001). Specificity of signal peptide recognition in tat-dependent bacterial protein translocation. Journal of bacteriology, 183(2), 604–610. doi:10.1128/JB.183.2.604-610.2001
          </p>
          <p>
          Dalia, T. N., Hayes, C. A., Stolyar, S., Marx, C. J., McKinlay, J. B., & Dalia, A. B. (2017). Multiplex genome editing by natural transformation (MuGENT) for synthetic biology in Vibrio natriegens. ACS synthetic biology, 6(9), 1650-1655.
          </p>
          <p>
          Jong, W. S., Vikstrom, D., Houben, D., van den Berg van Saparoea, H. B., de Gier, J. W., & Luirink, J. (2017). Application of an E. coli signal sequence as a versatile inclusion body tag. Microb Cell Fact, 16(1), 50. doi:10.1186/s12934-017-0662-4
          </p>
          <p>
          Lee, P. A., Tullman-Ercek, D., & Georgiou, G. (2006). The bacterial twin-arginine translocation pathway. Annual review of microbiology, 60, 373–395. doi:10.1146/annurev.micro.60.080805.142212
          </p>
          <p>
          Synthetic Genomics, Inc. (2017). VmaxTM Express Electrocompetent Cells User Guide. La Jolla, CA: Synthetic Genomics.
          </p>
          <p>
          Malherbe, G., Humphreys, D. P., & Davé, E. (2019). A robust fractionation method for protein subcellular localization studies in Escherichia coli. BioTechniques, 66(4), 171-178. doi:10.2144/btn-2018-0135
          </p>
        </div>
      </div>
    </div>
    </div>



    <!--button grid-->


    <div class="homegrid">
      <div class="homegrid-item" id="homegrid-text" style="background-image: url(bgpics/022.jpg); background-size: cover;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Description">PROJECT</a></div>
      <div class="homegrid-item" id="homegrid-text" style="background-image: url(bgpics/011.jpg); background-size: cover;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Lab">LAB</a></div>
      <div class="homegrid-item" id="homegrid-text" style="background-image: url(bgpics/033.jpg); background-size: cover;"><a href="https://2019.igem.org/Team:Aalto-Helsinki/Human_Practices">HUMAN<br>OUTREACH</a></div>
    </div>



    <!--PARTNERS-->

    <div id="partners">
      <div class="row">
        <div class="column1">
          <img class="photo" src="logos/dark/Aalto_BIZ_black.png">
        </div>
        <div class="column1">
          <img class="photo" src="logos/dark/Aalto_CHE_EN_black.png">
        </div>
        <div class="column1">
          <img class="photo" src="logos/dark/Aalto_ELEC_EN_black.png">
        </div>
        <div class="column1">
          <img class="photo" src="logos/dark/Aalto_ENG_EN_black.png">
        </div>
        <div class="column1">
          <img class="photo" src="logos/dark/Aalto_SCI_EN_black.png">
        </div>
        <div class="column1">
          <img class="photo" src="logos/dark/df_black.png">
        </div>
        <div class="column1">
          <img class="photo" src="logos/dark/Helsinki.png">
        </div>
        <div class="column1">
          <img class="photo" src="logos/dark/supported-by-hilife.png">
        </div>
        <div class="column1">
          <img class="photo" src="logos/dark/Kemianteollisuus_black.png">
        </div>
        <div class="column1">
          <img class="photo" src="logos/dark/LOIMU.jpg">
        </div>
        <div class="column1">
          <img class="photo" src="logos/dark/ThermoFisher_black.jpg">
        </div>


      </div>
    </div>

    <!--FOOTER-->
    <div class="footer">
      <a class="icon" id="fb" href="https://www.facebook.com/AaltoHelsinki"><i class="fab fa-facebook-square"></i></a>
      <a class="icon" id="twi" href="https://twitter.com/AaltoHelsinki"><i class="fab fa-twitter"></i></a>
      <a class="icon" id="ig" href="https://instagram.com/aaltohelsinki_igem"><i class="fab fa-instagram"></i></a>
      <a class="icon" id="blog" href="http://blog.aaltohelsinki.com"><i class="fab fa-tumblr-square"></i></a><br>
      Kemistintie 1, Espoo, Finland<br>
      team@aaltohelsinki.com
    </div>

  </body>
</html>
